assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -25,7 +25,7 @@
 </div>
 
 
-<div class="row" ng-if="!chromeless">
+<div class="row" ng-if="(!chromeless)">
```

Comment:
I've gotten into the habit of parens in an `ng-if` when the var is `!var` to make it a little more explicit.  The `!` is easy to miss otherwise.  Similar idea to `(function() { })()` vs `function() {}();`